packages/core/src/linter/featureSchema.ts

```diff
@@ -143,6 +147,8 @@ export function getFeatureZodSchema(
         message: "object is not flat",
       },
     ),
+    z.null(),
+    z.undefined(),
```

Comment:
keep only `null`, and not `undefined`?

if we keep only `null`, then it clashes with SDK's existing return type though.